source/board/nina_b1.c

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file    nina_b1.c
+ * @brief   board ID for the u-blox NINA-B1 EVA maker board
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2009-2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "target_config.h"
+
+const char *board_id = "1235";

```

Comment:
I sent an email to Mihail to assign two new board ids. One for EVK-NINA-B1 and one for this other board here.